### PR TITLE
[JENKINS-36975] Only render Admin link if logged in

### PR DESCRIPTION
**Pull request by imeredith**
# Description
 Only render Admin link if logged in

See [JENKINS-36975](https://issues.jenkins-ci.org/browse/JENKINS-36975).

# Submitter checklist
- [x] Link to JIRA ticket in description, if appropriate.
- [x] Change is code complete and matches issue description
- [x] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [x] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.
- [x] Ran Acceptance Test Harness against PR changes.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

@jenkinsci/code-reviewers @reviewbybees

### CODE REVIEW

**Comment by kzantow** [comment]
:bee: seems ok

File: blueocean-web/src/main/js/main.jsx [hunk 1]
```diff
@@ -37,14 +37,15 @@ class App extends Component {
     }
 
     render() {
+        const renderAdminLink = AppConfig.getInitialUser() !== "anonymous";
```
Comment (i386) [inline blueocean-web/src/main/js/main.jsx]: Shouldn't it check if the user can actually go to the manage jenkins screen?

File: blueocean-web/src/main/js/main.jsx [hunk 2]
```diff
@@ -37,14 +37,15 @@ class App extends Component {
...
         return (
             <div className="Site">
                 <header className="Site-header">
                     <div className="global-header">
                         <Extensions.Renderer extensionPoint="jenkins.logo.top"/>
                         <nav>
                             <Link to="/pipelines">Pipelines</Link>
-                            <a href="#">Administration</a>
+                            { renderAdminLink && <a href="#">Administration</a> }
```
Comment (i386) [inline blueocean-web/src/main/js/main.jsx]: Also this is missing the link to the manage page

Comment (imeredith) [inline blueocean-web/src/main/js/main.jsx]: Yes i just reread ticket and noticed this.

File: blueocean-web/src/main/js/main.jsx (hunk 1 above)
Comment (imeredith) [inline blueocean-web/src/main/js/main.jsx]: Well, it wasnt in scope of ticket, and I don't have a good way currently to do this. We need permissions stuff really before we do this. I could hack it probabl though if it is critical.

**Pull request closed without merging**